contrib/opencensus-ext-stackdriver/opencensus/ext/stackdriver/stats_exporter/__init__.py

```diff
@@ -110,18 +113,14 @@ def default_monitoring_labels(self):
         return self._default_monitoring_labels
 
 
-class StackdriverStatsExporter(base_exporter.StatsExporter):
+class StackdriverStatsExporter(object):
```

Comment:
We'll update `StatsExporter` after making the same changes to the prometheus exporter. This is an API change.